content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
+  }
+}
+```
+
+### Sample Response (public provider)
+
+```json
+{
+  "data": {
+    "id": "prov-fZn7uHu99ZCpAKZJ",
+    "type": "registry-providers",
+    "attributes": {
+      "name": "aws",
+      "namespace": "hashicorp",
+      "registry-name": "public",
+      "created-at": "2020-07-09T19:36:56.288Z",
+      "updated-at": "2020-07-09T20:16:20.538Z",
+      "permissions": {
+        "can-delete": true
+      }
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "my-organization",
+          "type": "organizations"
+        }
+      }
+    },
+    "links": {
+      "self": "/api/v2/organizations/my-organization/registry-providers/public/hashicorp/aws"
+    }
+  }
+}
+```
+
+## Delete a Provider
+
+`DELETE /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name`
+
+### Parameters
+
+| Parameter            | Description                                                                                                                                                                                                  |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `:organization_name` | The name of the organization to delete a provider from. The organization must already exist, and the token authenticating the API request must belong to the "owners" team or a member of the "owners" team. |
+| `:namespace`         | The namespace of the provider that will be deleted.                                                                                                                                                          |
+| `:name`              | The name of the provider that will be deleted.                                                                                                                                                               |
+| `:registry_name`     | Indicates whether this is a publicly maintained provider or private. Must be either `public` or `private`.                                                                                                                                                                                                      |
+
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
+
+| Status  | Response                  | Reason                                                      |
+| ------- | ------------------------- | ----------------------------------------------------------- |
+| [204][] | Nothing                   | Success                                                     |
+| [403][] | [JSON API error object][] | Forbidden - public provider curation disabled               |
+| [404][] | [JSON API error object][] | Provider not found or user not authorized to perform action |
+
+
+### Sample Request (private provider)
```

Comment:
Can you go through and capitalize all of the things in parentheses (I see we have some places where public provider isn't capitalized). We capitalize everything in titles, so that's why i'm asking for this.

```suggestion
### Sample Request (Private Provider)
```